### PR TITLE
Update main.yml

### PR DESCRIPTION
**Pull request by GeoffreyTien**
Config for container_hierarchy plugin

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -159,8 +159,13 @@ openio_oioswift_filter_container_hierarchy:
   account_first: true
   strip_v1: true
   swift3_compat: true
-  create_dir_placeholders: true
-  recursive_placeholders: true
+#  create_dir_placeholders: true
+#  recursive_placeholders: true
+  sentinel_hosts:
```
Comment (vincent-legoll) [inline defaults/main.yml:164]: ca serait pas mieux dans le role redis, tout ça ?

File: defaults/main.yml [hunk 2]
```diff
@@ -159,8 +159,13 @@ openio_oioswift_filter_container_hierarchy:
...
+    "{{ groups['openio_redis_cluster'] \
+    | map('extract', hostvars, ['openio_bind_address'])\
+    | join(':6012,') }}:6012"
+  sentinel_name: "{{ openio_oioswift_sds_proxy_namespace }}-master-1 "
```
Comment (cdelgehier) [inline defaults/main.yml:168]: Use `openio_oioswift_namespace` instead of `openio_oioswift_sds_proxy_namespace`

File: defaults/main.yml [hunk 3]
```diff
@@ -159,8 +159,13 @@ openio_oioswift_filter_container_hierarchy:
   account_first: true
   strip_v1: true
   swift3_compat: true
-  create_dir_placeholders: true
-  recursive_placeholders: true
+#  create_dir_placeholders: true
+#  recursive_placeholders: true
+  sentinel_hosts:
+    "{{ groups['openio_redis_cluster'] \
```
Comment (cdelgehier) [inline defaults/main.yml:165]: Replace this jinja by a default value like `192.168.1.1:6012,192.168.1.2:6012,192.168.1.3:6012`
Playbook will map extract the needed value.

File: defaults/main.yml [hunk 4]
```diff
@@ -159,8 +159,13 @@ openio_oioswift_filter_container_hierarchy:
   account_first: true
   strip_v1: true
   swift3_compat: true
-  create_dir_placeholders: true
-  recursive_placeholders: true
+#  create_dir_placeholders: true
```
Comment (cdelgehier) [inline defaults/main.yml:162]: why leave that there?

**Review by cdelgehier — CHANGES_REQUESTED** [review]

**Pull request closed without merging**